docs/root/compiled-binaries.md

```diff
@@ -4,10 +4,13 @@ hide_title: true
 hide_table_of_contents: true
 ---
 
-## Download compiled binary
```

Comment:
Sadly this title did not show in the sidebars (and it's level wasn't always appropriate) so I had to remove it from this common file to put it each time this file is used.